### DISCUSSION TITLE
My Setup code is locked

### DISCUSSION THREAD

**Discussion by JioseftheOwlman** [Pryaxis/TShock#2862] [Q&A]
Hello, I'm new here, today I made my terraria server and I need to access the setup code that is inside Files / tshock, but this .txt file ends in setup.lock, it should be noted that inside terraria when I enter the /setup command I get It says that it is disabled and to ask for help on this site

another thing is that within the game 
it asks me for permission for almost everything, invoke events, bosses, and other things

![setup locked](https://user-images.githubusercontent.com/119817497/205516735-e1be8b8d-90f9-4126-98f0-3709f52d7e4b.PNG)

**Comment by sgkoishi** [comment]
You might want to read [this](https://github.com/Pryaxis/TShock/discussions/2325#discussioncomment-772279).
